src/Controller/TiqrAppApiController.php

```diff
@@ -132,14 +120,14 @@ public function tiqr(UserAgentMatcherInterface $userAgentMatcher, Request $reque
         if ($operation === 'register') {
             $this->logger->notice(
                 'Got POST with registration response',
-                array('notificationType' => $notificationType, 'notificationAddress' => $notificationAddress)
+                ['notificationType' => $notificationType, 'notificationAddress' => $notificationAddress]
             );
             return $this->registerAction($userAgentMatcher, $request, $notificationType, $notificationAddress);
         }
         if ($operation === 'login') {
```

Comment:
candidate for const

Reply:
Given this is only used for logging. I'm not putting in the effort here. Has little business value IMHO